src/flag_gems/runtime/register.py

```diff
@@ -0,0 +1,96 @@
+from typing import Optional
+
+import torch
+
+from . import backend, commom_utils, error
+from .backend.device import DeviceDetector
+
+aten_lib = torch.library.Library("aten", "IMPL")
+
+
+class Register:
+    def __init__(
+        self,
+        config: Optional[tuple[tuple]],
+        user_unused_ops_list: Optional[list[str]] = [],
```

Comment:
we could set the default value of user_unused_ops_list as None.

Reply:
Done